packages/website/pages/files.js

```diff
@@ -231,104 +231,112 @@ export default function Files({ user }) {
     }
 
     return (
-      <tr className="bg-white bb">
-        <td data-label="Date" className="nowrap" title={nft.created}>
+      <tr className="bg-white border-b">
+        <td data-label="Date" className="whitespace-nowrap" title={nft.created}>
           {/* {nft.created.split('T')[0]} */}
           {formatTimestamp(nft.created)}
         </td>
-        <td data-label="Label" className="nowrap" title={nft.label}>
-          {!isRenaming ? (
-            <div
-              className={clsx(
-                'flex items-center justify-start',
-                renameError.length > 0 && 'text-w3storage-red'
-              )}
-            >
-              <span className="flex-auto">{renamedValue || nft.name}</span>
-              {renameError.length > 0 && (
+        <td data-label="Label" className="whitespace-nowrap" title={nft.label}>
+          <div className="flex justify-between items-center truncate ...">
+            {!isRenaming ? (
+              <div
+                className={clsx(
+                  'flex items-center justify-start ml-auto lg:ml-0 truncate ...',
+                  renameError.length > 0 && 'text-w3storage-red'
+                )}
+              >
                 <span
-                  className="rounded-full border-w3storage-red border flex-none w-6 h-6 flex justify-center items-center"
-                  title={renameError}
+                  className="flex-auto truncate ..."
```

Comment:
**todo:** Remove "..."

Reply:
it's part of tailwind classes. https://tailwindcss.com/docs/text-overflow#truncate

Reply:
**discussion:** It just looks like that; it's their convention that means "the rest of your classes"; it's confusing in this case specifically because truncate truncates with "..." cf: 

`<p class="normal-case ...">The quick brown fox ...</p>`
https://tailwindcss.com/docs/text-transform

Reply:
I read you, for the CID it won't even get used anymore and file names got commented out so let's proceed.